pkgs/tools/misc/yubikey-manager-qt/default.nix

```diff
@@ -52,6 +52,8 @@ in stdenv.mkDerivation rec {
     buildPythonPath "$pythonPath"
 
     wrapProgram $out/bin/ykman-gui \
+      --prefix LD_PRELOAD : "${yubikey-personalization}/lib/libykpers-1.so" \
```

Comment:
Not quite sure if macOS will be ever supported but this would help in case:

```suggestion
      --prefix LD_PRELOAD : "${yubikey-personalization}/lib/libykpers-1.${stdenv.hostPlatform.extensions.sharedLibrary}" \
```

Reply:
mhm. Nevermind. MacOS has no LD_PRELOAD anyway.

Reply:
Is it really required after all? Should LD_LIBRARY_PATH not be sufficient?

Reply:
Yes it worked without `LD_PRELOAD`. The PR has been updated.